### ISSUE TITLE
Discuss: Structure of tech coops list

### ISSUE THREAD

**Issue by hng** [hng/tech-coops#58]
As interest in tech-coops seems to be growing and this list is getting more exposure in the last few weeks, e.g. it was referenced in a [MIT press paper](https://data-feminism.mitpress.mit.edu/pub/a1ao95xs/release/3), I would like to put some work again into it.

The structure of this list has been mostly untouched. Especially the table/list of tech-coops could use some more structure, as it grows. 

I see three options:

1) Group list by regions/countries. Caveat: There are quite a number of tech coops working remotely and spanning over multiple regions/countries.
2) Group list by work areas, e.g. web development, game development. Caveat: Most tech-coops are generalist software/web development firms.
3) Both?

I think I would prefer 1), as it's the easiest and logical one. 

It would also be nice to have a interactive website at some point that allows filtering of the list of tech-coops, but I also like to have the list as a simple markdown file in Github.

What do you think?

**Comment by motorto** [comment]
Yeah I agree with you , on the choose of the structure (number one) . One possible way to circumvent the issue is specifically telling the ones that do work on a remote basis.


------------------

For example:

#### Country A

* Coop A 
* Coop B ***
* Coop C 


#### Country B

* Coop A ***
* Coop B 


*** Coops that do work on a a remote basis as well ! 

-------------------

Of course the symbol can be changed for example a world emoji don't know : )

**Comment by trivigy** [comment]
One additional point of interest would be to provide the legal documents some of these companies use to form and govern their respective cooperatives. If you have any already, wouls be greatly appreciated if you could add some.

**Comment by carlos-reynosa** [comment]
Just FYI guys, this repo could have github discussions enabled in order to help facilitate discussions like this:

* https://docs.github.com/en/discussions

**Comment by decentral1se** [comment]
1. sounds good! Glad to see this getting the attention it deserved!

**Comment by hng** [comment]
@motorto Thats a great suggestion! I think we should do it that way sort by regions/countries and add a note for remote coops.

@carlos-reynosa I've enabled discussions

I hope to have a bit more time to get started on this in February, but of course you could also just start restructuring the table ;)

**Comment by decentral1se** [comment]
If you need co-maintainers then let's fire up that cooperation :heart:

**Comment by bzz** [comment]
Breakdown by countries could be a good start. 

Makes me wonder if, in the long term and for maintainability reasons, it would make sense to have a separate .json/.md file per-coop, with some mandatory schema and a script that would generate a README.md from that + a template. (e.g did something simple like that for [bzz/computing-history](https://github.com/bzz/computing-history))

Also, do you think it makes sense to go though and merge all open PRs first, before re-structuring?

**Comment by motorto** [comment]
I agree with @bzz , maybe a system that allows easy insertion / removal of coops instead of having one huge file. Maybe create separated files by country, instead of individual ones per coop.

The merging of PRs is something that should be done before the cooperation starts (the ones that @hng  think are okay to ), at least in my opinion.

I would totality cooperate with this ! If you need any help with something say so !

**Comment by shonfeder** [comment]
I agree that having entries in separate files is preferable. Otherwise we're headed into a constant merge conflict territory (see #78).

An alternative to, or an additional measure on top of, putting all the disparate entries into the README.md would be generating a gh-pages hosted static site using the entries. This is trivial to set up, and I'd be happy to lend a hand.

A more ambitious alternative/addition would be aiming to migrate this directory to use something like the [murmurations protocol](https://murmurations.network/). That's a whole different ball of wax, of course, and orthogonal to this more immediate problem. But I thought worth mentioning :)
